jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/DatasourceDataFetcher.java

```diff
@@ -18,7 +18,6 @@
 import java.util.Set;
 
 public class DatasourceDataFetcher {
-
```

Comment:
Actually I tend to have this empty line after the class declaration... Googled it a lot about and all guides speaks differently about it. We need to set our own standards around it, and for me it just looks better with empty line, at least in InteliJ :)